### DISCUSSION TITLE
Cpu usage increased

### DISCUSSION THREAD

**Discussion by uthisra** [AzuraCast/AzuraCast#5606] [General]
Hi
im using 17.2 stable , no stereo tools 
i have a 10 cpu server with 12 stations , after update i see CPU usage is very High,
if i restart thats down to 15% then slowly build up to 100%
Please Advice

**Comment by Sakretsos** [comment]
Hello @uthisra this is a bug that occures after the last stable - release version.
Developers is searching why this is happens!
I think they will fix this issue soon!
I advice you to use stable version 0.17.1
